common/web/types/src/util/compiler-interfaces.ts

```diff
@@ -42,6 +42,8 @@ export interface CompilerCallbacks {
   loadLdmlKeyboardTestSchema(): Buffer;
   reportMessage(event: CompilerEvent): void;
   loadKvksJsonSchema(): Buffer;
+  loadKpjJsonSchema(): Buffer;
+  forceDirectories(dir: string): void;
```

Comment:
From only the name, I was unclear that  `forceDirectories` creates directories. What about something like `forceCreateDirectories`?

Reply:
I'm going to pull this back out of here, because it's used by kmc, which has fs access anyway.

Reply:
This will be done in a follow-up PR